src/store.js

```diff
@@ -24,6 +24,31 @@ export const CONFIG_UPDATED = 'config.updated';
 export const EFFECTIVE_GENOME_UPDATED = 'effective.genome.updated';
 export const STORE_DESTROYED = 'store.destroyed';
 
+export function expKeyStatesHas(keyStates, stateName, key, prefix) {
```

Comment:
Can you add a comment to this function? The purpose isn't clear from its name.

Reply:
Sure